docs/sources/developers/plugins/create-a-grafana-plugin/_index.md

```diff
@@ -1,27 +1,18 @@
 ---
-description: How-to topics for plugin development
-title: Create a plugin
+title: Create a Grafana plugin
+menuTitle: Create a plugin
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - documentation
+description: An index of how-to topics for Grafana plugin development.
+weight: 300
 ---
 
 # Create a Grafana plugin
 
-This section contains how-to topics for developing Grafana plugins.
+This section contains how-to topics for developing and extending Grafana plugins.
 
-- [Build a Grafana plugin](https://grafana.github.io/plugin-tools/docs/creating-a-plugin)
-  - [Build a panel plugin](https://grafana.com/tutorials/build-a-panel-plugin/)
-  - [Build a data source plugin](https://grafana.com/tutorials/build-a-data-source-plugin/)
-  - [Build a data source backend plugin](https://grafana.com/tutorials/build-a-data-source-backend-plugin/)
-  - [Build a logs data source plugin]({{< relref "../build-a-logs-data-source-plugin.md">}})
-  - [Build a streaming data source plugin]({{< relref "../build-a-streaming-data-source-plugin.md">}})
-- Extend a Grafana plugin
-  - [Add annotations]({{< relref "add-support-for-annotations.md">}})
-  - [Add anonymous usage reporting]({{< relref "add-anonymous-usage-reporting.md">}})
-  - [Add authentication for a data source plugin]({{< relref "add-authentication-for-data-source-plugins.md">}})
-  - [Add Explore queries]({{< relref "add-support-for-explore-queries.md">}})
-  - [Add query editor help]({{< relref "add-query-editor-help.md">}})
-  - [Add variables]({{< relref "add-support-for-variables.md">}})
-  - [Create panel option editors]({{< relref "custom-panel-option-editors.md">}})
-- [Sign a plugin]({{< relref "sign-a-plugin.md">}})
-- [Automate development with CI](https://grafana.github.io/plugin-tools/docs/ci)
-- [Create nested plugins](https://grafana.github.io/plugin-tools/docs/nested-plugins)
-- [Extend configurations](https://grafana.github.io/plugin-tools/docs/advanced-configuration)
+- [Develop a plugin]- {{< relref "./develop-a-plugin/_index.md" >}}
```

Comment:
Why is this not just `{{< section menuTitle="true">}}` like the migration guide? Currently it renders poorly
<img width="865" alt="Screenshot 2023-06-27 at 17 40 05" src="https://github.com/grafana/grafana/assets/19860021/14c07013-d694-4b2e-a486-5d3479415a59"> The suggested code fixes it

Reply:
Fixed.